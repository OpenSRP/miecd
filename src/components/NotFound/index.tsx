import * as React from 'react';

class NotFound extends React.Component {
    public render() {
        return <div>404!</div>;
    }
}

export default NotFound;
