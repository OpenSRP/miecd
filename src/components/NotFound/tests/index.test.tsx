import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from '..';

describe('NotFound', () => {
    it('renders correclty', () => {
        const wrapper = mount(<NotFound />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
