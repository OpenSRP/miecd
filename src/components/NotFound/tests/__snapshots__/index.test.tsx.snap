// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correclty 1`] = `
<NotFound>
  <div>
    404!
  </div>
</NotFound>
`;
